x/crosschain/types/keys.go

```diff
@@ -26,6 +26,8 @@ const (
 	// CCTXIndexLength is the length of a crosschain transaction index
 	CCTXIndexLength          = 66
 	MaxOutboundTrackerHashes = 5
+
+	UsableRemainingFeesPercentage = uint64(95)
```

Comment:
how is this value decided, maybe we can add a comment to explain?

Reply:
This was the previous value, used . 
I added some comments on how it is used , but I dont think there is a specific reason for this number ,  it more like we use a number less than 100 just to be safe in the calcuations so as not to overmint